include/aspect/heating_model/adiabatic_heating.h

```diff
@@ -0,0 +1,94 @@
+/*
+  Copyright (C) 2011 - 2015 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef __aspect__heating_model_adiabatic_heating_h
+#define __aspect__heating_model_adiabatic_heating_h
+
+#include <aspect/heating_model/interface.h>
+#include <aspect/simulator_access.h>
+
+namespace aspect
+{
+  namespace HeatingModel
+  {
+    using namespace dealii;
+
+    /**
+     * A class that implements a standard adiabatic heating rate.
+     *
+     * This adds the term from adiabatic compression heating
+     *    $ \alpha T (\mathbf u \cdot \nabla p) $
```

Comment:
you need to write "\\" every place you have a single "\" right now. This has nothing to do with latex but because of c++ escaping rules.